src/onedb/local/5.6.0_to_5.7.8.rb

```diff
@@ -0,0 +1,58 @@
+require 'set'
+require 'base64'
+require 'zlib'
+require 'pathname'
+require 'yaml'
+require 'opennebula'
+
+$: << File.dirname(__FILE__)
+
+include OpenNebula
+
+module Migrator
+    def db_version
+        "5.6.0"
+    end
+
+    def one_version
+        "OpenNebula 5.6.0"
+    end
+
+    def up
+        feature_2253()
+
+        return true
+    end
+
+    private
+
+    def feature_2253()
+        @db.transaction do
```

Comment:
Do we need to update VMs? Maybe we can relay on the current default in LibvirtKVM.cc. Test. A VM in poweroff can be resume to the same state after upgrade. verify please